_drafts/offensive-programming.md

```diff
@@ -0,0 +1,599 @@
+---
+title: Defensive Programming - Friend or Foe?
+description:
+  Embedded systems can benefit from defensive and offensive programming with the
+  use of asserts, timeouts, and watchdogs.
+author: tyler
+image: img/offensive-programming/cover.png
+---
+
+One of the worst things about embedded development (especially in C) is
```

Comment:
FWIW I think the slightly shorter introduction you had before was better (but not a big deal).